pyproject.toml

```
@@ -0,0 +1,69 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mera"
+authors = [{ name = "QuakeCoRE" }]
+description = "Mixed-effects regression analysis"
+readme = "README.md"
+requires-python = ">=3.11"
+dynamic = ["version"]
+dependencies = [
+  "numpy",
+  "pandas",
+  "pymer4 @ git+https://github.com/ucgmsim/pymer4.git@get_ranef_cond_std",
+  "natsort",
+]
+
+[project.optional-dependencies]
+test = [
+  "pytest",
+]
+
+[tool.deptry.per_rule_ignores]
+DEP002 = ["pytest"]
```

```suggestion
[project.optional-dependencies]
test = [
  "pytest",
]
```

The per-rule-ignores are unnecessary because you can simply designate  the `test` optional dependency group as optional. See the suggestion to the CI action.